spring-integration-core/src/test/java/org/springframework/integration/config/InvalidPriorityChannelParserTests.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2002-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.config;
+
+import org.hamcrest.Matchers;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.springframework.beans.factory.parsing.BeanDefinitionParsingException;
```

Comment:
Here is my STS import config - it automatically adds a blank line between groups:

![screen shot 2016-04-29 at 11 49 28 am](https://cloud.githubusercontent.com/assets/483832/14921512/a16b9c42-0e00-11e6-8ad1-f2112d33dee8.png)